revolv/payments/models.py

```diff
@@ -78,6 +87,13 @@ def reinvestments(self, admin=None, project=None, queryset=None):
             queryset = queryset.filter(admin=admin).order_by('created_at')
         if project:
             queryset = queryset.filter(project=project).order_by('created_at')
+        if start_date and not end_date:
```

Comment:
Same thing here. In fact, this code is repeated exactly from above (and again down on line 275). Can you extract it into a utility?